src/SketchField.jsx

```diff
@@ -257,41 +257,44 @@ class SketchField extends PureComponent {
     if (e) e.preventDefault();
     let { widthCorrection, heightCorrection } = this.props;
     let canvas = this._fc;
+    let backgroundCanvas = this._fbc;
     let { offsetWidth, clientHeight } = this._container;
     let prevWidth = canvas.getWidth();
     let prevHeight = canvas.getHeight();
     let wfactor = ((offsetWidth - widthCorrection) / prevWidth).toFixed(2);
     let hfactor = ((clientHeight - heightCorrection) / prevHeight).toFixed(2);
     canvas.setWidth(offsetWidth - widthCorrection);
     canvas.setHeight(clientHeight - heightCorrection);
-    if (canvas.backgroundImage) {
-      // Need to scale background images as well
-      let bi = canvas.backgroundImage;
-      bi.width = bi.width * wfactor;
-      bi.height = bi.height * hfactor
-    }
-    let objects = canvas.getObjects();
-    for (let i in objects) {
-      let obj = objects[i];
-      let scaleX = obj.scaleX;
-      let scaleY = obj.scaleY;
-      let left = obj.left;
-      let top = obj.top;
-      let tempScaleX = scaleX * wfactor;
-      let tempScaleY = scaleY * hfactor;
-      let tempLeft = left * wfactor;
-      let tempTop = top * hfactor;
-      obj.scaleX = tempScaleX;
-      obj.scaleY = tempScaleY;
-      obj.left = tempLeft;
-      obj.top = tempTop;
-      obj.setCoords()
-    }
+    backgroundCanvas.setWidth(offsetWidth - widthCorrection);
```

Comment:
These 2 operations can also go inside the forEach, or not?

Reply:
Yep, I'll try

Reply:
It's works :ok_hand: